packages/next-swc/crates/next-api/src/server_actions.rs

```diff
@@ -192,11 +219,11 @@ async fn get_actions(module: Vc<Box<dyn Module>>) -> Result<Vc<ModuleActionMap>>
 /// Our graph traversal visitor, which finds the primary modules directly
 /// referenced by [parent].
 async fn get_referenced_modules(
-    parent: Vc<Box<dyn Module>>,
-) -> Result<impl Iterator<Item = Vc<Box<dyn Module>>> + Send> {
-    primary_referenced_modules(parent)
+    (layer, module): (ActionLayer, Vc<Box<dyn Module>>),
```

Comment:
Why isn't that just 2 arguments

Reply:
This is invoked by `NonDeterministic::visit`'s `fn` impl, which only accepts a single arg.